### ISSUE TITLE
composeUp fails in 0.14.6

### ISSUE THREAD

**Issue by ASravanthi** [avast/gradle-docker-compose-plugin#313]
Hi,

After upgrading com.avast.gradle.docker-compose plugin version from 0.14.5 to 0.14.6, we are getting error.

Gradle Version: 7.0.2

------
build.gradle (sub-module **foo**)

plugins {
id "com.avast.gradle.docker-compose" version "0.14.6"
}

dockerCompose {
    useComposeFiles = ['docker-compose.yml']
    noRecreate = true
    isRequiredBy(tasks.run)
    startedServices = ["s1", "s2"]
    stopContainers = false
}

------
build.gradle (sub-module **bar**)

task run(overwrite: true, type: JavaExec) {
    if (!project.hasProperty('noDocker')) {
        dependsOn project(':foo').tasks.composeUp
    }
}

tasks.clean.finalizedBy project(':foo').tasks.composeDownForced

-------

Getting following error on ./gradlew clean build on root project. Project has only 2 modules foo and bar.

```
An exception occurred applying plugin request [id: 'com.avast.gradle.docker-compose', version: '[0.14.6]']
> Failed to apply plugin 'com.avast.gradle.docker-compose'.
   > Could not create an instance of type com.avast.gradle.dockercompose.ComposeExtension.
      > Unable to determine constructor argument #1: missing parameter of type Project, or no service of type Project.
```

**Comment by augi** [comment]
Hello, thank you for reporting this issue! It could be a problem when applying the plug-in using _DSL syntax_. Could you please try to use the _traditional_ syntax, as shown in the README? :pray:

**Comment by ASravanthi** [comment]
@augi tried with traditional syntax as in README. v0.14.6 gave same error as above.

**Comment by augi** [comment]
Could you also please try to use the latest Gradle 7.x version? :pray: If it didn't help, are you able to deliver a minimal reproducible example?

**Comment by augi** [comment]
I've just successfully reproduced the error. Looks like an error in Gradle 7.0.2. Gradle 7.1 and later works well. Anyway, I will immediately release a new version of this plugin that is able to overcome this issue.

**Comment by augi** [comment]
Released as `0.14.7`.

**Comment by ASravanthi** [comment]
Issue is resolved with v0.14.7. Thanks for quick turnaround.
